.gitignore

```
@@ -6,3 +6,5 @@ coverage/
 coverage_report/
 **/*.jl.*.cov
 .vscode/
+examples/*.tec
+examples/*.mesh
```

```suggestion
examples/**/*.tec
examples/**/*.mesh
```
To make it work for subfolders as well?